code/modules/vending/vending.dm

```diff
@@ -2185,3 +2186,131 @@
 
 		product_list += new/datum/data/vending_product(/obj/item/sponge/cheese, 2, hidden=1)
 
+/obj/machinery/vending/air_vendor
+	name = "Oxygen Vending Machine"
+	desc = "Here, you can buy the oxygen that you need to live."
+	icon_state = "O2vend"
+	icon_panel = "O2vend-panel"
+	icon_off = "O2vend-off"
+	icon_broken = "O2vend-broken"
+	icon_fallen = "O2vend-fallen"
+	can_hack = FALSE
+	pay = TRUE
+	acceptcard = TRUE
+	vend_delay = 0
+	slogan_list = list("Come get a breath of fresh air",
+	"You NEED this to live!.",
+	"Breathing is GOOD!",
+	"Contains only 2% farts!")
+	var/global/image/holding_overlay_image = image('icons/obj/vending.dmi', "O2vend-slot")
+
```

Comment:
air_vendor used to have different deconstruct flags
`deconstruct_flags = DECON_CROWBAR | DECON_WRENCH | DECON_MULTITOOL`
These seem to fit better with lack of `can_hack` thus `DECON_SCREWDRIVER` - >`DECON_CROWBAR`